### DISCUSSION TITLE
Bluetooth trouble on Intel 9260 + XPS 13

### DISCUSSION THREAD

**Discussion by hung0702** [sebanc/brunch#786] [Q&A]
Anyone know if the Intel 9260 is giving me problems or something else? If it's the wireless card, could you guide me in the direction of fixing it?

System: XPS 13 9360
Grub2WIn: 2.2.0.7
Wireless: Intel 9260
ChromeOS: v87.0.7280.109
Brunch: r87 stable 20201227
rammus_recovery: 87 (13505.73.087.0.4280.109)

In crosh, I couldn't get 'connectivity' to display anything. I guess it needs some cmd or arg but I can't tell from the help documentation. 'connectivity help' was similarly uninterpretable. I saw elsewhere that someone's older Intel-WiFi laptop was having Bluetooth issues that resolved by putting the system to sleep and waking it. I tried closing the lid for 15 min, but that did not resolve the situation. I rebooted within ChromeOS, but that did not resolve it either. In Windows, Bluetooth continues to work as expected.

The Bluetooth toggle "works" and begins scanning, but never picks anything up. I have also tried "option=iwlwifi_backport" and it rebuilt successfully, but the Bluetooth issue remained (so I reverted).

**Comment by Warrentheo** [comment]
Most likely this is an issue that has been resolved by Intel in the Windows drivers, but has not been corrected in the Linux kernel yet...  Intel WiFi and Bluetooth are usually on the same piece of hardware, but are presented as two separate devices to the system...  In the case of Brunch, the Linux Kernel comes from Brunch, and is where this driver comes from, and so updating Brunch may resolve this issue now or maybe in the future...  I also would continue to look for some of the Brunch boot options that are being put in to see if any apply to your system...  The only other option is to break out a code editor and correct it yourself...
